charts/victoria-metrics-operator/values.yaml

```diff
@@ -30,6 +30,8 @@ image:
   pullPolicy: IfNotPresent
 
 crds:
+  # -- manages CRD creation. Disables CRD creation only in combination with `crds.plain: false` due to helm dependency conditions limitation
```

Comment:
Can we throw a error directly for  `crds.enabled: false` and `crds.plain: true` then, to prevent incorrect usage? It should be fine since the default `crds.plain` is false.

Reply:
comma in `condition` implements `OR` operation

Reply:
so it will not help

Reply:
yeah, I mean to add an additional check somewhere ourselves, for `crds.enabled: false` with`crds.plain: true`.

Reply:
updated